CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `ModeSolverData.pol_fraction` and `ModeSolverData.pol_fraction_waveguide` properties to compute polarization fraction of modes using two different definitions.
 - `ModeSolverData.to_dataframe()` and `ModeSolverData.modes_info` for a convenient summary of various modal properties of the computed modes.
+- `tidy3d.converter.py` to convert .lsf project files into Tidy3D files, as well as `test.test_package.example.lsf` to test it. 
```

Comment:
since users will be reading this, would be best to describe how it could be used by the user. for example
```
- conversion from .lsf project files into Tidy3D files through command line: `tidy3d convert example.lsf tidy3d_script.py`
```